src/game/Game.java

```diff
@@ -107,98 +103,164 @@ public int getPlayerTurn() {
 		return playerTurn;
 	}
 
+	public Owner getOwner() {
+		return Owner.values()[(getPlayerTurn() - 1)];
+	}
+
+	public Owner getOtherOwner() {
+		if (getPlayerTurn() == 1) {
+			return Owner.values()[1];
+		} else {
+			return Owner.values()[0];
+		}
+	}
+
```

Comment:
Useful for checking owners of pieces compared to the player whose turn it is. Mainly used in checking if a piece is able to move.